geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -155,7 +167,7 @@
                         <phase>package</phase>
                         <configuration>
                             <attach>true</attach>
-                            <finalName>geomesa-accumulo-${project.version}</finalName>
+                            <finalName>geomesa-accumulo_${scala.binary.version}-${project.version}</finalName>
```

Comment:
fyi- see now that hardcoding in this PR isn't constrained to `project.artifactId` and `parent.project.artifactId`.